variables.tf

```diff
@@ -811,6 +811,12 @@ variable "kong_portal_enabled" {
   default     = "on"
 }
 
+variable "kong_portal_api_enabled" {
+  description = "(Optional) Define whether or not the Kong Portal API should be enabled."
+  type        = string
+  default     = "on"
+}
```

Comment:
From the description it seems like this should be a Boolean. Kong might expect a string of "on" or "off", but we can set that value based on the Boolean, which would provide validation even without the custom validation defined

Reply:
This was a boolean until me and @johncuthbertuk had a discussion around consistency - the other variables;

```variable "kong_vitals_enabled" {
  description = "Define whether or not Kong Vitals should be enabled."
  type        = string
}
variable "kong_portal_enabled" {
  description = "Define whether or not the Kong Portal should be enabled."
  type        = string
}
```

Are also strings and not bools - we can move everything to bools but it would likely introduce a breaking change

Reply:
As discussed with @dan-hill2802 , variable validation & other tech debt items to be done under a separate ticket, AS-4690